### PR TITLE
drop i18n folder (outdated)

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: i18n/translations.qrc [hunk 1]
```diff
@@ -1,47 +0,0 @@
-<RCC>
```
Comment (m-kuhn) [inline i18n/translations.qrc:1]: This file might still be needed

Comment (m-kuhn) [reply i18n/translations.qrc:1]: Or generated

**Comment by m-kuhn** [comment]
It looks like someone copied .qm files from qt into the source code here
@m-kuhn why did you do that?

**Comment by m-kuhn** [comment]
No idea ... we should probably take those files from the installation or remove them from the resource file

**Comment by nirvn** [comment]
FYI, we should dynamically generate the translations.qrc to avoid missing translations (see the Russian discovery in PR #2186 )

**Comment by 3nids** [comment]
superseded by https://github.com/opengisch/QField/pull/2190

**Pull request closed without merging**